README.md

```diff
@@ -0,0 +1,112 @@
+![sttp-model](https://github.com/softwaremill/sttp-openai/raw/master/banner.png) Banner should be changed to sttp openAi
+
+[![Join the chat at https://gitter.im/softwaremill/sttp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/softwaremill/sttp)
+[![CI](https://github.com/softwaremill/sttp-openai/workflows/CI/badge.svg)](https://github.com/softwaremill/sttp-openai/actions?query=workflow%3ACI+branch%3Amaster)
+
+[//]: # ([![Maven Central]&#40;https://maven-badges.herokuapp.com/maven-central/com.softwaremill.sttp&#41;&#40;https://maven-badges.herokuapp.com/maven-central/com.softwaremill.sttp&#41;)
+
+sttp is a family of Scala HTTP-related projects, and currently includes:
```

Comment:
Let's just write a short introduction, that sttp openai uses sttp client to describe requests to OpenAI endpotins or sth like that